### ISSUE TITLE
Lots of spam with INFO error after last update (version 2.71.14)

### ISSUE THREAD

**Issue by BartekSz95** [MeridiusIX/Modular-Encounters-Systems#297]
After last update (version 2.71.14), my server is spamming every time with this INFO log:
```
20:43:30.0179 [INFO]   Keen: MES Spawner / Error: System.NullReferenceException: Object reference not set to an instance of an object.
   at ModularEncountersSystems.Behavior.Subsystems.Trigger.TriggerSystem.ProcessCommandReceiveTriggerWatcher(Command receivedCommand)
   at System.Action`1.Invoke(T obj)
   at ModularEncountersSystems.Sync.ChatManager.ProcessServerChat(ChatMessage chatData)
   at ModularEncountersSystems.Sync.ChatManager.ChatReceivedNetwork(ChatMessage chatData)
   at ModularEncountersSystems.Sync.SyncManager.NetworkMessageReceiver(UInt16 handlerId, Byte[] initialData, UInt64 sender, Boolean fromServer)
```
Some "in action" screens:
![obraz](https://github.com/MeridiusIX/Modular-Encounters-Systems/assets/79801491/7a6d9c4e-95ca-4f47-822a-967da3434d94)
![obraz](https://github.com/MeridiusIX/Modular-Encounters-Systems/assets/79801491/f8a34abb-03b6-44e6-b939-c06d0acb45c5)

**Comment by CptArthur** [comment]
I potentially fixed this issue with 2.71.15; Could you check if it is still happening?

**Comment by BartekSz95** [comment]
Actually I don't see any issues and errors on logs created by MES. Thanks for fast fix! :)
